### ISSUE TITLE
rtMedia compatibility with PHP 7.4.1 and other Notices/Warnings.

### ISSUE THREAD

**Issue by SID177** [rtMediaWP/rtMedia#1553]
```
PHP Deprecated:  array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead in /var/www/htdocs/wp-content/plugins/buddypress-media/app/main/controllers/template/RTMediaNav.php on line 238
```
- Remove `array_key_exists( 'media_type', $rtmedia_query->action_query )`

```
PHP Notice:  Trying to access array offset on value of type null in /var/www/htdocs/wp-content/plugins/buddypress-media/app/main/controllers/template/rtmedia-functions.php on line 318, 503, 82, 180, 246, .....
```
- Replace `if ( $rtmedia_backbone['backbone'] ) {` with `if ( ! empty( $rtmedia_backbone['backbone'] ) ) {` in rtmedia-functions.php on every occurrences.

- Need to update `buddypress-media/lib/getid3` (https://www.getid3.org/) library. PHP Deprecated warning occurs on video upload.

Need to add all the Warnings and Notices.

**Comment by pooja-muchandikar** [comment]
@SID177 

Tested with `develop` branch and with the fixes no notices/warnings were shown in console nor in the debug.log.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.0`
